python/cuml/cuml/ensemble/randomforest_common.pyx

```diff
@@ -545,3 +598,57 @@ class BaseRandomForestModel(Base, InteropMixin):
                 align_bytes=align_bytes,
             )
         return fil_model
+
+    @property
+    def oob_score_(self):
+        """Out-of-bag score for the training dataset.
+
+        Only available if oob_score=True.
+
+        Returns
+        -------
+        score : float
+            Score of the training dataset using an out-of-bag estimate.
+            For classification, this is the accuracy score.
+            For regression, this is the R-squared score.
+        """
+        if not self.oob_score:
+            raise AttributeError(
```

Comment:
What was the thinking behind having a custom exception text here instead of raising a standard `AttributeError`?

Reply:
Are you saying that we want the text to be something like:
"has not attribute oob_score_"?